build_win32.sh

```diff
@@ -4,14 +4,14 @@ set -eu
 rm -rf Tor/
 rm -rf Data/
 
-curl -fsSL "https://dist.torproject.org/torbrowser/7.5.4/tor-win32-$TOR_VERSION.zip" -o tor-win32-$TOR_VERSION.zip
-curl -fsSL "https://dist.torproject.org/torbrowser/7.5.4/tor-win32-$TOR_VERSION.zip.asc" -o tor-win32-$TOR_VERSION.zip.asc
+curl -fsSL "https://dist.torproject.org/torbrowser/7.5.4/tor-win32-$TOR_WIN32_VERSION.zip" -o tor-win32-$TOR_WIN32_VERSION.zip
```

Comment:
I would prefer just waiting for the tor team to release the expert binaries for Windows, instead of maintaining two different versions of tor binaries across different OS.

Reply:
That's not likely to happen for another few weeks, so since there's no urgent security issues at hand I think what you've done here is fine.